commands_add.go

```diff
@@ -46,10 +46,31 @@ func (a addCmd) ToAuditEntry() string {
 		<username>%s</username>
 		<funds>%.02f</funds>
 	</userCommand>`,
-		time.Now().UnixNano()/1e6, redisBaseKey, a.id, a.userID, a.amount.ToFloat(),
+		time.Now().UnixNano()/1e6, redisBaseKey, a.txID, a.userID, a.amount.ToFloat(),
 	)
 }
 
 func (a addCmd) Execute() {
-	consoleLog.Warning("Not implemented: ADD")
+	// Create an account if one does not exist
+	if !accountStore.HasAccount(a.userID) {
+		consoleLog.Noticef("Creating account for %s", a.userID)
+		if err := accountStore.CreateAccount(a.userID); err != nil {
+			consoleLog.Error(err.Error())
+			abortTx("Account creation failed")
+		}
+	}
+
+	// Add the amounts
+	consoleLog.Infof("Adding %s to %s", a.amount, a.userID)
+	userAccount := accountStore.GetAccount(a.userID)
+	if userAccount == nil {
+		consoleLog.Errorf("Internal account creation error for %s", a.userID)
+		abortTx("Internal account creation error")
+	}
+
+	consoleLog.Debugf("Old balance for %s is %s", a.userID, userAccount.Balance)
+
+	userAccount.AddFunds(a.amount)
+
+	consoleLog.Infof("New balance for %s is %s", a.userID, userAccount.Balance)
 }
```

Comment:
You can let the user / log know the transaction finished by adding something like [this](https://github.com/DistributedDesigns/worker/blob/master/commands_quote.go#L61)